controller/src/main/java/io/pravega/controller/server/ControllerServiceStarter.java

```diff
@@ -233,6 +251,7 @@ protected void startUp() {
             setController(new LocalController(controllerService, serviceConfig.getGRPCServerConfig().get().isAuthorizationEnabled(),
                     serviceConfig.getGRPCServerConfig().get().getTokenSigningKey()));
 
+            CompletableFuture<Void> eventProcessorFuture = CompletableFuture.completedFuture(null); 
```

Comment:
no, completed future is correct. If eventProcessor is not enabled then this is set. Otherwise eventProcessorFuture is set to future returned by `EventProcessors.bootStrap`